pkg/kubectl/sorting_printer.go

```diff
@@ -60,8 +61,35 @@ func (s *SortingPrinter) sortObj(obj runtime.Object) error {
 	if len(objs) == 0 {
 		return nil
 	}
+
+	sorter, err := SortObjects(objs, s.SortField)
+	if err != nil {
+		return err
+	}
+
+	switch list := obj.(type) {
```

Comment:
Can you add a comment about how `runtime.SetList(obj, objs)` doesn't work for `v1.List` because its items are `runtime.RawExtension`, which are not `runtime.Objects`?